Sample.java

```diff
@@ -1,7 +1,124 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// ## Problem1 (https://leetcode.com/problems/sort-colors/)
+
+// Time Complexity : O(n)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : yes
+// Any problem you faced while coding this : I didnt face any problem while coding this
+
+class Solution {
+    public void sortColors(int[] nums) {
+        int n = nums.length;
+        int low = 0;
+        int high = n-1;
+        int mid = 0;
+        while(mid <= high){
+            if(nums[mid] == 2){
+                swap(nums, mid, high);
+                high--;
+            }else if(nums[mid] == 0){
+                swap(nums, mid, low);
+                mid++;
+                low++;
+            }else{
+                if(nums[mid] == 1){
+                    mid++;
+                }
+            }
+        }   
+        }
+        private void swap(int[] nums, int i, int j){
+            int temp = nums[i];
+            nums[i] = nums[j];
+            nums[j] = temp;
+        }
+
+}
+// ## Problem2 (https://leetcode.com/problems/3sum/)
+
+// Time Complexity : O(n ^ 2 log n)

```

Comment:
Check your complexity here . Your complexity is exponential . For all the solutions of three sum , none of the solution gives exponential complexity